odc/stats/utils.py

```diff
@@ -132,6 +132,36 @@ def bin_annual(
     return tasks
 
 
+def bin_rolling_annual(
+    cells: dict[tuple[int, int], Cell],
+    temporal_range,
+    years: int,
+    interval: int,
+) -> dict[tuple[str, int, int], list[CompressedDataset]]:
+    binner = rolling_season_binner(
+        mk_rolling_years_rules(temporal_range, years, interval)
+    )
+
+    tasks = {}
+    for tidx, cell in cells.items():
+        # This is a great pylint warning, but doesn't apply here because we
```

Comment:
Can't this entire warning+suppression be avoided by removing `utc_offset` and putting `cell.utc_offset` inside the lambda?